src/android/com/microsoftopentechnologies/intellij/helpers/DatePickerCellEditor.java

```diff
@@ -129,9 +127,31 @@ public Object getCellEditorValue() {
     }
 
     private static void checkAccess(int var0) {
-        if(System.getSecurityManager() != null && !Modifier.isPublic(var0)) {
+        if (System.getSecurityManager() != null && !Modifier.isPublic(var0)) {
             throw new SecurityException("Resource is not accessible");
         }
     }
 
+    private static void checkPackageAccess(Class var0) {
+        checkPackageAccess(var0.getName());
+    }
+
+    private static void checkPackageAccess(String var0) {
+        SecurityManager var1 = System.getSecurityManager();
+        if (var1 != null) {
+            String var2 = var0.replace('/', '.');
+            int var3;
+            if (var2.startsWith("[")) {
+                var3 = var2.lastIndexOf(91) + 2;
+                if (var3 > 1 && var3 < var2.length()) {
+                    var2 = var2.substring(var3);
+                }
+            }
+
+            var3 = var2.lastIndexOf(46);
+            if (var3 != -1) {
+                var1.checkPackageAccess(var2.substring(0, var3));
+            }
+        }
+    }
```

Comment:
Why does this code have variables names such as `var0`, `var1` etc? Is this decompiled code from somewhere?